### ISSUE TITLE
the h3_postgis extension doesn't install for PG11 and lower

### ISSUE THREAD

**Issue by robe2** [zachasme/h3-pg#108]
I'm packaging h3-pg for windows and in my tests, the h3-postgis extension fails install on PG11 with this error:

```
ERROR: syntax error at or near "AGGREGATE"
```

I think the issue is with the h3_postgis--4.0.2--4.0.3.sql script and h3_postgis--4.0.3.sql, both use

```
CREATE OR REPLACE AGGREGATE
```

Which wasn't introduced until PostgreSQL 12.  These should be replaced with just CREATE AGGREGATE for lower than 12.  Or I guess you can just skip supporting PG 11 for main branch.

I didn't check the other releases to see if those have issues.

**Comment by zachasme** [comment]
If this is the only thing keeping `h3_postgis` from running on PG11, then I think we should replace it with just "CREATE AGGREGATE" as you suggest.

How are you packaging the extension for windows? [I just added Windows support to the main branch three weeks ago](https://github.com/zachasme/postgresql-extensions-cmake/actions/workflows/test-windows.yml), and I am very interested to hear if it is working for users.

**Comment by robe2** [comment]
I'm using the main branch and compiling against mingw64 (ninja, my new chain). This had no dependencies so was easy to do with my new gcc 12 chain which I'm in middle of porting PostGIS and all it's dependencies to.

Listed here - https://postgis.net/windows_downloads/

For the PG11 release, since in theory no one will be upgrading from lower, I just hand-fixed the  h3_postgis--4.0.3.sql file.
